package exceptions;

public class NestedTryCatchTester {

	public static void main(String[] args) {
		 NestedTryCatch.calculate(10, 2);

	}

}
